docs/source/examples/Natural and artificial perturbations.ipynb

```diff
@@ -11,7 +11,16 @@
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "gzip was not found on your system! You should solve this issue for astroquery.eso to be at its best!\n",
```

Comment:
Interesting, I was not aware of this issue on Windows. Let's leave it like that anyway.

Reply:
Yes I installed 7zip, but the error still shows.